locales/en.default.json

```diff
@@ -107,6 +107,8 @@
   "products": {
     "product": {
       "add_to_cart": "Add to cart",
+      "update_cart": "Update cart",
+      "added_to_cart": "Added to cart",
```

Comment:
@scottgmeadows @nicklepine Let me know if this content is approved and I can request translations

Reply:
👀 👋  I think the word "added" is the most confusing to me from my first impression with the prototype for the success and cart aware state. `Added to cart` is more communicating a success than informing how many items are in the cart.

Looking at some of the benchmark Nick shared more broadly, seems like more generic wording fit better with the 3 quantity input roles + some visual difference in the appearance of the button.

1. Quantity you want to add to your cart = `Add to cart`
2. You added this amount of product in your cart = `In cart`
3. You are changing the number of items available in your cart = `Update cart`

I really wanted to understand where I had my discomfort so I quickly and briefly [explored something](https://screenshot.click/27-24-9sucl-rhitw.png) if it can help assess my feedback/observation. 🙏